frontend/package.json

```diff
@@ -1,77 +1,15 @@
 {
-  "name": "frontend",
-  "private": true,
-  "version": "0.0.0",
-  "type": "module",
+  "name": "figuringout",
+  "version": "1.0.0",
   "scripts": {
-    "dev": "vite",
-    "build": "tsc && vite build",
-    "lint": "eslint ./src/ --report-unused-disable-directives --max-warnings 0",
-    "lint-fix": "eslint ./src/ --fix",
-    "format": "prettier --write \"**/*.{js,jsx,ts,tsx,json,md}\"",
-    "preview": "vite preview",
-    "prepare": "cd .. && husky install frontend/.husky"
+    "dev": "concurrently --prefix \"[{name}]\" --names \"backend,frontend\" -c \"blue,green\" \"cd backend && npm run dev \" \"cd frontend && npm run dev\" ",
+    "install:all": "concurrently --prefix \"[{name}]\" --names \"backend-install,frontend-install\" -c \"blue,green\" \"cd backend && npm install \" \"cd frontend && npm install\" "
   },
-  "lint-staged": {
-    "*.ts": "eslint",
-    "*.tsx": "eslint"
-  },
-  "husky": {
-    "hooks": {
-      "pre-commit": "lint-staged && pretty-quick --staged"
-    }
+  "license": "MIT",
+  "devDependencies": {
+    "concurrently": "^8.2.2"
   },
   "dependencies": {
-    "@aadeshk/medium-common": "^1.0.2",
-    "@radix-ui/react-dropdown-menu": "^2.1.1",
-    "@radix-ui/react-slot": "^1.1.0",
-    "@types/quill": "^2.0.14",
-    "animate.css": "^4.1.1",
-    "axios": "^1.6.8",
-    "canvas-confetti": "^1.9.3",
-    "class-variance-authority": "^0.7.0",
-    "clsx": "^2.1.1",
-    "lodash.debounce": "^4.0.8",
-    "lucide-react": "^0.397.0",
-    "react": "^18.2.0",
-    "react-chartjs-2": "^5.2.0",
-    "react-confetti": "^6.1.0",
-    "react-dom": "^18.2.0",
-    "react-quill": "^2.0.0",
-    "react-router-dom": "^6.23.0",
-    "react-toastify": "^10.0.5",
-    "react-use": "^17.5.0",
-    "tailwind-merge": "^2.3.0",
-    "tailwind-scrollbar-hide": "^1.1.7",
-    "tailwindcss-animate": "^1.0.7",
-    "typewriter-effect": "^2.21.0",
     "@react-oauth/google": "^0.12.1"
-  },
-  "devDependencies": {
-    "@eslint/js": "^9.3.0",
-    "@types/lodash.debounce": "^4.0.9",
-    "@types/node": "^20.12.12",
-    "@types/react": "^18.2.66",
-    "@types/react-dom": "^18.2.22",
-    "@typescript-eslint/eslint-plugin": "^7.2.0",
-    "@typescript-eslint/parser": "^7.2.0",
-    "@vitejs/plugin-react": "^4.2.1",
-    "autoprefixer": "^10.4.19",
-    "eslint": "^8.57.0",
-    "eslint-config-prettier": "^9.1.0",
-    "eslint-plugin-prettier": "^5.1.3",
-    "eslint-plugin-react": "^7.34.1",
-    "eslint-plugin-react-hooks": "^4.6.0",
-    "eslint-plugin-react-refresh": "^0.4.6",
-    "globals": "^15.3.0",
-    "husky": "^8.0.3",
-    "lint-staged": "^15.2.5",
-    "postcss": "^8.4.38",
-    "prettier": "^3.2.5",
-    "pretty-quick": "^4.0.0",
-    "tailwindcss": "^3.4.3",
-    "typescript": "^5.2.2",
-    "typescript-eslint": "^7.10.0",
-    "vite": "^5.2.0"
```

Comment:
Please verify your changes before pushing your code. This isn't the expectation.

Closing this PR because it has more problems than desired outcome.